### DISCUSSION TITLE
Inline code-formatting broken when cut off in long commit messages

### DISCUSSION THREAD

**Discussion by markus-codewrangler** [community/community#61657] [Repositories]
### Select Topic Area

Bug

### Body

Even though the last word of the commit message is wrapped in back-ticks, the stub of the word is not formatted as inline code.

"Commits" tab in a PR:
![image](https://github.com/community/community/assets/115093421/aefe3beb-505f-4c59-a795-4d89e7fec70b)

View commit details:
![image](https://github.com/community/community/assets/115093421/ec77ffbe-7f9f-4138-a7ab-5b35ade537d4)

For clarification an example of correct inline code formatting (`node-fetch`, `fetch`):
![image](https://github.com/community/community/assets/115093421/cb524c65-2c57-494c-9015-7c5b87b69904)

**Comment by moonfury-ops** [comment]
Thank you
